k8s/utils/health/cluster_health_check.py

```diff
@@ -0,0 +1,89 @@
+from kubernetes import client, config
```

Comment:
@harshvkarn -- Add the copyright header to this file, followed by what is the functionality/need for this function. Do we need to use a python client or can we use kubectl with json path to get the same functionality achieved?

Reply:
@kmova pushed a commit with licence header and Functionality info. Currently, we are using the Kubernetes Python client, but we can even use the kubectl and can achieve the same using ansible playbook.